sormas-api/src/main/java/de/symeda/sormas/api/contact/ContactDto.java

```diff
@@ -197,9 +197,13 @@ public class ContactDto extends SormasToSormasShareableDto {
 	@Size(max = FieldConstraints.CHARACTER_LIMIT_DEFAULT, message = Validations.textTooLong)
 	private String tracingAppDetails;
 	private ContactProximity contactProximity;
+	@HideForCountriesExcept
 	@SensitiveData
 	@Size(max = FieldConstraints.CHARACTER_LIMIT_DEFAULT, message = Validations.textTooLong)
 	private String contactProximityDetails;
```

Comment:
is the proximity details field also something specific to Germany ?
i don't se this in the ticket